res/values/strings.xml

```diff
@@ -679,6 +679,12 @@
     <string name="qrscan_contains_url">Scanned QR code URL:\n\n%1$s</string>
     <string name="qrscan_fingerprint_label">Fingerprint</string>
     <string name="qrscan_x_verified_introduce_myself">%1$s verified, introduce myself…</string>
+    <string name="withdraw_verifycontact_explain">This QR code can be scanned by others to setup a contact with you.\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
```

Comment:
```suggestion
    <string name="withdraw_verifycontact_explain">This QR code can be scanned to get in contact with you.\n\nIt can be withdrawn at any time, and revived by scanning it again.</string>
```

Reply:
I don't understand what "reviving" a code does, nor who is able to do it?
In my head it is withdrawn so that scanning it does nothing, but all of a sudden it can be revived that way…?
How is it ever withdrawn then?

Reply:
Hm, maybe this needs some more thought then, if it wasn't clear even to you as a technical person.

It's:
- First, _You_ create a group invite QR code.
- then, _others_ can scan it to join the group
- Now, _you_ scan it to withdraw it, after confirming that you want to withdraw it. `withdraw_verifygroup_explain` is the confirmation & explanation that's shown at this point.
- then, when _others_ scan it now, trying to join your group, it does nothing.
- Now, _you_ scan it again to revive it. `revive_verifygroup_explain` is the confirmation & explanation that's shown it this point.
- then, _others_ can scan it again to join your group.

Withdrawing contact invite codes works the same way.

Is it clear now? I guess you want to change your suggested changes again? Esp. can you say what other formulation would have made it clear?

Reply:
Ah, so _creators_ can scan it to rescind it. Understood.